src/System.Data.SqlClient/tests/ManualTests/SQL/SqlBulkCopyTest/Helpers.cs

```diff
@@ -55,6 +49,19 @@ public static int TryExecute(DbCommand cmd, string strText)
             return cmd.ExecuteNonQuery();
         }
 
+        public static int TryExecuteNonQueryAzure(string strConnectionString, string strCommand, int commandTimeout = 60)
```

Comment:
Not: Usually methods with name Try* return a bool to signify if the execution was a successful one.

Reply:
Point taken, the reasoning behind naming it as such was going by the naming convention already present in other methods `Helpers.cs`. I have updated the method name to `ExecuteNonQueryAzure`